elytron/WFLY-17143-request-uri-parameters.adoc

```diff
@@ -0,0 +1,138 @@
+= Add the ability to specify that the OIDC Authentication Request should include request and request_uri parameters
+:author:            Prarthona Paul
+:email:             prpaul@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+OpenID Connect is an authentication mechanism that builds on OAuth 2.0 and allows a user to login to a web application using credentials established by an OpenID provider. Authentication requests sent using OpenID Connect can be signed and optionally encrypted. This can be achieved using 2 auth request parameters: `request` and `request_uri`, which are both optional. 
```

Comment:
We should mention in the overview that the `elytron-oidc-client` subsystem doesn't currently include the ability to specify the `request` or `request_uri` in the authentication request.

Reply:
fixed!